jobs_detector/main.py

```diff
@@ -1,32 +1,86 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
 import click
 import requests
 from bs4 import BeautifulSoup
+from .settings import BASE_URL, BASE_DIR
+from .exceptions import InvalidURLException
 
-from jobs_detector import settings
-
-DEFAULT_KEYWORDS = [
-    # set some default keywords here
-]
 
+DEFAULT_KEYWORDS = ['Remote', 'Postgres', 'Python', 'Javascript','React','Pandas']
 
 @click.group()
 def jobs_detector():
     pass
 
-
 @jobs_detector.command()
-@click.option('-i', '--post-id', type=str, required=True)
+@click.option('-i', '--post-id', type=str, required=True, help='[required]')
 @click.option('-k', '--keywords', type=str, default=','.join(DEFAULT_KEYWORDS))
 @click.option('-c', '--combinations', type=str,
               callback=lambda _, x: x.split(',') if x else x)
-def hacker_news(post_id, keywords, combinations):
+
+def hacker_news(post_id, keywords, combinations=None):
+    keywords = [word for word in keywords.split(',')]
```

Comment:
No need to use a list comprehension here. `keywords.split(',')` should return a list